.gitignore

```diff
@@ -4,3 +4,7 @@ npm-debug.log
 .DS_Store
 .eslintrc.js
 docs
+stub_api_key.txt
+stub_project_id.txt
+stub_default_credentials.json
+config.tar
```

Comment:
It would be better to generate these files in a directory and exclude that directory instead of listing all the files individually.